src/main/java/com/java_academy/gui/Main.java

```diff
@@ -0,0 +1,28 @@
+package com.java_academy.gui;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+import java.io.File;
+import java.net.URI;
+
+public class Main extends Application {
+
+    @Override
+    public void start(Stage primaryStage) throws Exception{
+        URI url=new File("src/main/java/com/java_academy/gui/board.fxml").toURI();
```

Comment:
You could load this with `FXMLLoader loader = new FXMLLoader(getClass().getResource("/board.fxml"));`. You have to put `board.fxml` in `/src/main/resources` for this to work.